packages/kolibri-common/components/SearchBox.vue

```diff
@@ -184,8 +200,16 @@
     padding: 0;
     padding-left: 8px;
     margin: 0;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
     vertical-align: middle;
     border: 0;
+    @media (max-width: 1260px) {
+      &::placeholder {
+        font-size: 12px;
```

Comment:
Lets remove this font-size, as is being too small, and the font-size of the rest of the text doesnt change, so I think the placeholder text shouldnt neither.